.gitignore

```diff
@@ -28,5 +28,4 @@ ledger/
 tests/snapshots-tmp/
 
 # Directory storing the ethereum build output for tests run
-tests/.test_dependencies/ethereum/*
-!tests/.test_dependencies/ethereum/.ethereum_application_build_goes_there
+tests/.test_dependencies/*
```

Comment:
@tdejoigny-ledger @fbeutin-ledger the idea to keep `.ethereum_application_build_goes_there` was to guide the user in putting the binaries at the proper place for tests.

However it also mean we keep folders in `tests/.test_dependencies` for versioning...

Can we do : 
```
tests/.test_dependencies/*
!tests/.test_dependencies/ethereum/.ethereum_application_build_goes_there
```

So that only the file is kept ?

Reply:
if Vscode extension is used, it's seemless for the user. He doesn't need to know where the ETH binaries are stored. 
So is it really useful to keep this file ?

And we could even change gitignore to:
`tests/.test_dependencies`

Reply:
**if** the user is using the vscode extension. I think we should not assume what the user will do...

Reply:
Ok updated